fig/Shell Script For Loop Flow Chart.svg

```diff
@@ -0,0 +1,1083 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
This file is called `Shell Script For Loop Flow Chart.svg`, but referenced as `shell_script_for_loop_flow_chart.svg`. Please change the file name